twint/tweet.py

```diff
@@ -114,10 +114,11 @@ def Tweet(tw, config):
     t.trans_dest = ''
     if config.Translate == True:
         try:
-            ts = translator.translate(t.tweet)
+            # config.Translate
+            ts = translator.translate(text=t.tweet, dest=config.TranslateDest)
             t.translate = str(ts.text)
             t.trans_src = str(ts.src)
             t.trans_dest = str(ts.dest)
-        except Exception as e:
+        except KeyError:
```

Comment:
You sure that `KeyError`is the right error to handle? Usually it happens with `dict` variables, my previously described way to handle exceptions was just an example and might be the right exception to handle